### ISSUE TITLE
Viewing multipane guide in a narrow layout doesn't show code snippets

### ISSUE THREAD

**Issue by evelinec** [OpenLiberty/openliberty.io#1003]
While viewing a multipane guide in a narrow layout, ie, a phone, or minimize the browser till everything shows up on a single pane, there's no code snippets showing up. 

Talked to @steven1046, he said the hotspot code snippets should show up. He's fixing the css that enables this capability. 

Perhaps we need some design discussion for this.

**Comment by steven1046** [comment]
Designs for reference: https://ibm.ent.box.com/folder/45309210116

**Comment by steven1046** [comment]
When we originally added the hotspots functionality to the guides, I had added code to insert the code snippets that the hotspot would highlight right after it. But i don't think that is the correct approach, as it looks like in the design it should show the full file when they hover over a hotspot. I'm talking to design to confirm first.

**Comment by steven1046** [comment]
"Regarding mobile behavior, if user clicks on hotspot, the code file would be shown with the lines highlighted.  user clicks *Back to text* button to return to hotspot.   I believe Megan intended the code to be shown by scrolling to it, but I'm not sure if that works when we have multiple files (in tabs on desktop).  the other way to do it when the hotspot is clicked is to show the file in a modal pane.  the user can scroll around the file but they can't scroll back into the instruction text.  they click *Back to text* to return.   if we do it that way, then perhaps have the Back to text button be floating and fixed in position if they scroll around the file"

**Comment by yeekangc** [comment]
The technical design and solution IMO shouldn't require any changes in the AsciiDoc.

**Comment by yeekangc** [comment]
@steven1046, please review the technical design and solution with @evelinec @gkwan-ibm when ready.

**Comment by steven1046** [comment]
I'm waiting on a code review, then I can show it off on the staging site.

**Comment by steven1046** [comment]
There are no extra asciidoc changes required.

**Comment by steven1046** [comment]
@yeekangc @evelinec @gkwan-ibm the mobile hotspots are on the staging site: https://openlibertydev.mybluemix.net/

**Comment by gkwan-ibm** [comment]
Not bad. My initial comments
1.  Any better wording for "Back to text" or a "return" image? (see point 4)
2. On the desktop, when I click on the "Table of contents" image, nothing happen. But, on my phone, the table of contents window shows up.
3. I tried on my phone. The box can be done better.
![IMG_4280](https://user-images.githubusercontent.com/29576299/54055123-05e30f80-41ba-11e9-8bbe-f6b9af4e6dba.jpg)

**Comment by steven1046** [comment]
For #2, when you click on a hotspot, we don't want the guide to be scrollable so we also disable opening the table of contents so they can't leave the hotspot code.

**Comment by mdenby** [comment]
For the narrow window or mobile view, the background of the actionable areas/words will be the gray/blue color. A part of the code will show under the actionable area and have a fade over it with an arrow. When the arrow is clicked the code window will expand. There is no 'close' arrow, the user will simply continue scrolling to the next section. The code can also be copied when not expanded with the copy icon. I've shown an example of the user clicking the '/health' code a paragraph to show the code--this is from Thuy-Hahn's updated highlighting. Finally, I've shown an example of the user scrolling back up--the TOC and top navigation will appear.

Also, the font for the code text is Google font Roboto Mono.

Please let me know if you need specific assets from me or if these jpgs are enough.

![mobile_full copy 6](https://user-images.githubusercontent.com/26823018/55365376-5ea08200-54b2-11e9-8e8c-24563e06158a.png)
![mobile_full copy 7](https://user-images.githubusercontent.com/26823018/55365377-5ea08200-54b2-11e9-9452-01ebea6890e5.png)
![mobile_full copy 8](https://user-images.githubusercontent.com/26823018/55365378-5ea08200-54b2-11e9-85c0-45723d0a607a.png)
![mobile_full copy 9](https://user-images.githubusercontent.com/26823018/55365379-5ea08200-54b2-11e9-9a4e-28343ff8ab3d.png)
![mobile_full copy 10](https://user-images.githubusercontent.com/26823018/55365380-5ea08200-54b2-11e9-81e8-2bd84da9a5d8.png)
![mobile_full copy 11](https://user-images.githubusercontent.com/26823018/55365381-5ea08200-54b2-11e9-94f3-a73524546321.png)
![mobile_full copy 12](https://user-images.githubusercontent.com/26823018/55365382-5ea08200-54b2-11e9-86ac-249cce7c23e6.png)
![mobile_full copy 13](https://user-images.githubusercontent.com/26823018/55365383-5ea08200-54b2-11e9-9659-aeaf57533cb2.png)

**Comment by mdenby** [comment]
Here's an example of the code panel in a table. Also, an example of when there are multiple tabs. 
![mobile_full copy 14](https://user-images.githubusercontent.com/26823018/55493401-ded7fc00-5606-11e9-9ca8-9b4119e6328a.png)
![mobile_full copy 16](https://user-images.githubusercontent.com/26823018/55493404-ded7fc00-5606-11e9-854c-a0f45aa6b344.png)

**Comment by steven1046** [comment]
@mdenby are there redlines for the styling for the hotspots and the code snippets that are not hotspots?

**Comment by steven1046** [comment]
Closing this as the initial implementation is done and i'm making a new issue to address the updates requested under #1217.
